ui/feral_druid/inputs.ts

```diff
@@ -0,0 +1,251 @@
+import { FeralDruid_Rotation_FinishingMove as FinishingMove } from '/tbc/core/proto/druid.js';
+import { FeralDruid_Options as DruidOptions } from '/tbc/core/proto/druid.js';
+import { RaidTarget } from '/tbc/core/proto/common.js';
+import { Spec } from '/tbc/core/proto/common.js';
+import { NO_TARGET } from '/tbc/core/proto_utils/utils.js';
+import { ActionId } from '/tbc/core/proto_utils/action_id.js';
+import { Player } from '/tbc/core/player.js';
+import { Sim } from '/tbc/core/sim.js';
+import { EventID, TypedEvent } from '/tbc/core/typed_event.js';
+import { IndividualSimUI } from '/tbc/core/individual_sim_ui.js';
+import { Target } from '/tbc/core/target.js';
+import { getEnumValues } from '/tbc/core/utils.js';
+import { ItemSlot } from '/tbc/core/proto/common.js';
+
+// Helper function for identifying whether 2pT6 is equipped, which impacts allowed rotation options
+function numThunderheartPieces(player: Player<Spec.SpecFeralDruid>): number {
+	const gear = player.getGear();
+	let numPieces = 0;
+	const itemIds = [31048, 31042, 31034, 31044, 31039];
+	getEnumValues(ItemSlot).map(slot => Number(slot)).forEach(slot => {
+		const equippedItem = gear.getEquippedItem(slot)
+		if (equippedItem == null) {
+			numPieces = numPieces;
+		}
+		else if (itemIds.includes(equippedItem!.item.id)) {
+			numPieces = numPieces + 1;
+		}
+	});
```

Comment:
You could shorten this with something like,

`gear.asArray().map(equippedItem => equippedItem?.item.id).filter(id => itemIds.includes(id)).length`